spec/bolt_server/transport_app_spec.rb

```diff
@@ -787,6 +795,44 @@ def post_over_transport(transport, action, body_content, multiple: false)
       end
     end
 
+    describe '/project_inventory_targets' do
+      let(:bolt_project) { { 'name' => 'my_project' } }
+      let(:targets) { %w[one two three] }
```

Comment:
Would it be worth including Nick's edge-case for `protocol` as a test case here to make sure that `transport` properly handles that? Thinking his example of

```
targets:
  - uri: ssh://foo.example.com
    config:
      transport: remote
```

where `protocol` would be `ssh`, but `transport` is `remote`.